sass/components/list.sass

```diff
@@ -0,0 +1,22 @@
+$list-border: 1px solid $border !default
+$list-background-color: $white !default
+$list-radius: $radius !default
+
+$list-item-border: 1px solid $border !default
+$list-item-hover-color: $background !default
+
+.list
+  background-color: $list-background-color
+  border: $list-border
+  border-radius: $list-radius
+  .list-item
+    color: black
+    cursor: default
+    display: flex
+    justify-content: flex-start
+    padding: 8px 16px
+    &:not(:last-child)
+      border-bottom: $list-item-border
+  & .is-hoverable:hover
```

Comment:
I like this idea. But the syntax is `&.is-hoverable:hover` (without the space) and should be nested under `.list-item`.

Also, can you add a `&.is-active` state?

Reply:
which color should it have? i tried to give it primary but then the text color is still black